doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,148 @@
+
+a|*Adding EXPECT results in test scenarios*
+
+* It represents the objects and their fields whose exact values are checked based on the provided information.
+* The *EXPECT* section defines the expected results based on the *GIVEN* input facts.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about adding EXPECT results in new test scenarios designer, see xref:test-designer-create-test-scenario-template-con[].
+* For more information about adding EXPECT results in test scenarios(legacy), see xref:test-scenarios-legacy-EXPECT-proc[].
+
+|*KIE session*
+
+KIE session can be set on test scenario level settings.
+
+|`+`
+|`+`
+|-
+
+a|*KIE base on test scenario level*
+
+You can set KIE base on test scenario level settings.
+|`-`
+|`+`
+|-
+
+a|*KIE base on project level*
+
+You can set KIE base on project level settings.
+|`+`
+|`+`
+|-
+
+|*Simulated date and time*
+|`-`
+|`+`
+|-
+
+a|*Rule flow group*
+
+You can specify a rule flow group to be activated so that all rules within that group can be tested.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about setting rule flow group in new test scenarios, see xref:test-designer-global-settings-panel-rule-based-proc[].
+* For more information about setting rule flow group in test scenarios(legacy), xref:test-scenarios-legacy-GIVEN-proc[].
+
+a|*Globals*
+
+* Global variables are named objects that are visible to the decision engine but are different from the objects for facts.
+* Setting global variables for new test scenario is deprecated .
+* In case you want to reuse data sets for different scenarios, you can use the *Background* instance.
+
+|`-`
+|`+`
+
+a|
+
+* For more information about *Background* instance in new test scenarios, see xref:test-scenarios-background-instance-con[].
+* For more information about global variables in test scenarios(legacy), see xref:test-scenarios-legacy-create-proc[].
+
+a|*CALL METHOD*
+
+* You can use this to invoke a method from another fact when the rule execution is initiated.
+* You can invoke any Java class methods from the Java library or from a JAR that was imported for the project.
+
+|`+`
+|`+`
+
+a|
+
+* For more information about calling a method in new test scenarios, see xref:test-designer-expressions-syntax-intro-ref[].
+* For more information about calling a method in test scenarios(legacy), see xref:test-scenarios-legacy-create-proc[].
+
+a|*Modify an existing fact*
+
+* You can modify a previously inserted fact in the decision engine between executions of the scenario.
+
+|`-`
+|`+`
+
+|For more information about modifying an existing fact in test scenarios(legacy), see xref:test-scenarios-legacy-GIVEN-proc[].
+
+a|*Bound variable*
+
+* Sets the value of the field to the fact bound to a selected variable.
+* In new test scenario designer, you can not define a variable inside test scenario grid and reuse it inside *GIVEN* or *EXPECTED* cells
```

Comment:
Suggested rewrite:

In the new test scenario designer, you can not define a variable inside a test scenario grid and reuse it inside *GIVEN* or *EXPECTED* cells.

 //added articles and period.